app/assets/stylesheets/base/typography.scss

```diff
@@ -96,27 +105,79 @@ body {
   h5,
   h6 {
     font-family: var(--ff-sans-serif);
-    margin: var(--content-rythm) 0;
-    margin-top: calc(var(--content-rythm) * 2);
+    margin: calc(var(--content-rhythm) * 2) 0 calc(var(--content-rhythm) / 2);
     padding: 0;
+
+    & + p,
+    & + ul {
+      margin-top: 0;
+    }
   }
 
   h1 {
-    font-size: 1.85em;
+    font-size: var(--fs-2xl);
```

Comment:
Something to think through, but it seemed like a fairly good idea to keep font-size relative (using `em`s) to the rest of the text. It's a special snowflake since we use variables elsewhere, but the nature of this area (content body), especially the fact that it's user-generated content and not really part of the UI controls itself (like widgets headers, buttons, etc) made me feel it was worth being a special snowflake.

It also makes things easier to handle in terms of responsiveness (see below, where you had to overwrite font-size for several breakpoints)

Reply:
Another argument for not using variables in this case is because these elements (and its styling) are being used also in comments body - and comments have smaller font size than the articles, so we'd have to overwrite these font-sizes again, only for comments, to keep the visual balance right.

Reply:
We chatted about this and decided to stick to raw `em` variables, but we can append each line with a comment to say which `var(--fs-XYZ)` it is for easy reference.